e2e/orchestrator_event_task_test.go

```diff
@@ -88,8 +88,10 @@ func testOrchestratorEventTask(executionStream pb.Execution_StreamClient, instan
 			require.NotEmpty(t, exec.Outputs.Fields["timestamp"].GetNumberValue())
 		})
 		t.Run("delete process", func(t *testing.T) {
-			_, err := client.ProcessClient.Delete(context.Background(), &pb.DeleteProcessRequest{Hash: processHash})
-			require.NoError(t, err)
+			lcdBroadcastMsg(processmodule.MsgDelete{
```

Comment:
these kinds of tests are weird, it looks like we are not testing anything and in that case, the whole test should be removed

Reply:
fixed in https://github.com/mesg-foundation/engine/pull/1758